plugins/wazo_fanvil/common/templates/base-V67.tpl

```diff
@@ -285,9 +285,12 @@
             <AutoOnhookTime>0</AutoOnhookTime>
         </port>
     </call>
-    <dsskey>
```

Comment:
This line was deleted but there is the closing tag at line 340. Do you want to remove it entirely or the `dsskey` tag should be re-added?

Reply:
indeed it is an error on my part, however I think that the tag is added automatically by the post because I did not have a functioning error. Can you add it please.